torch/csrc/jit/passes/python_print.cpp

```diff
@@ -122,6 +122,7 @@ const static std::unordered_set<std::string> reserved_names = {
     "aten",
     "attribute",
     "CONSTANTS",
+    "PICKLES",
```

Comment:
Where is this used?

Reply:
this was left in from a previous change, will remove